src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const logo = document.getElementsByClassName('logo')[0];
+const div = document.createElement('div');
+
+document.body.appendChild(div);
+
+const resolvedMessage = 'Promise was resolved!';
+const rejectionMessage = 'Promise was rejected!';
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve(resolvedMessage);
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(rejectionMessage);
+  }, 3000);
+});
+
+const promiseResolve = (message) => {
+  div.textContent = message;
+};
+
+const promiseReject = (message) => {
+  div.textContent = message;
+};
+
+promise1.then(promiseResolve).catch(promiseReject);
+promise2.then(promiseResolve).catch(promiseReject);
```

Comment:
Currently, both promises are handled separately. If you want to handle the first promise that resolves or rejects, consider using `Promise.race([promise1, promise2])` to race the two promises and handle the one that settles first. This is particularly useful if you want to display a message as soon as either the logo is clicked or the timeout occurs.